src/server/game/Handlers/GroupHandler.cpp

```diff
@@ -721,14 +721,15 @@ void WorldSession::HandleRaidReadyCheckOpcode(WorldPacket& recvData)
 
 void WorldSession::HandleRaidReadyCheckFinishedOpcode(WorldPacket & /*recvData*/)
 {
-    //Group* group = GetPlayer()->GetGroup();
-    //if (!group)
-    //    return;
+    Group* group = GetPlayer()->GetGroup();
```

Comment:
could you check if this opcode is handled in world thread or in map thread and if the Group related methods are threadsafe (readonly operations should be threadsafe) ?

Reply:
world thread